### PR TITLE
Design/details page

### PR DESCRIPTION
**Pull request by alanblee**
# Description
Added graphs for population and job panes. Still have a lot of polishing to do but have the general layout
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Change Status
- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback
## Has This Been Tested
- [ ] Yes
- [x] No
- [ ] Not necessary
## Does this require communication between DS and WEB?
- [ ] Yes, and I have already reached out to someone
- [ ] Yes, but I still need to reach out to someone
- [x] No
## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least two peers
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by berachele — APPROVED** [review]
This is looking good! Thanks for cleaning up that dry code

**Review by lyndsiWilliams — APPROVED** [review]
Looking good!

File: src/components/common/Graphs/graphType.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const lineGraph = (set1, set2, set3) => {
       color: "#000"
     },
     showlegend: true,
-    // title: set1.graphName,
+    title: set1.graphName ? set1.graphName : "",
```
Comment (ddhorstman) [inline src/components/common/Graphs/graphType.js:44]:
```suggestion
    title: set1.graphName ??  "",
```

File: src/components/common/Graphs/graphType.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ export const barGraph = (set1, set2, set3) => {
   let dataPlot = [trace1, set2 ? trace2 : {}, set3 ? trace3 : {}];
 
   let layout = {
-    // title: set1.graphName,
+    title: set1.graphName ? set1.graphName : "",
```
Comment (ddhorstman) [inline src/components/common/Graphs/graphType.js:100]:
```suggestion
    title: set1.graphName ?? "",
```

**Review by ddhorstman — CHANGES_REQUESTED** [review]
This is definitely an improvement! I can see your design works with the loading skeletons which is awesome. However I'm noticing a major issue - your new Population and Jobs panes tend to render on top of each other, making them unreadable. I see it specifically in single-city detail view. It seems to be related somehow to changing selectedCities. You should fix that before we merge this in.

**Review by ddhorstman — APPROVED** [review]
The graph logic isn't working well here, but that issue is also present in master. The next order of business should be fixing that, but for now, these changes are good!

**Pull request merged**